sql/core/src/test/resources/sql-tests/results/charvarchar.sql.out

```diff
@@ -663,6 +663,478 @@ Location [not included in comparison]/{warehouse_dir}/char_part
 Partition Provider  	Catalog
 
 
+-- !query
+create table str_tbl(c string, v string) using parquet
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+insert into str_tbl values
+    (null, null),
+    (null, 'S'),
+    ('N', 'N '),
+    ('Ne', 'Sp'),
+    ('Net  ', 'Spa  '),
+    ('NetE', 'Spar'),
+    ('NetEa ', 'Spark '),
+    ('NetEas ', 'Spark'),
+    ('NetEase', 'Spark-')
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+create table char_tbl4(c7 char(7), c8 char(8), v varchar(6), s string) using parquet
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+insert into char_tbl4 select c, c, v, c from str_tbl
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+select c7, c8, v, s from char_tbl4
+-- !query schema
+struct<c7:string,c8:string,v:string,s:string>
+-- !query output
+N      	N       	N 	N
+NULL	NULL	NULL	NULL
```

Comment:
The indentations do not match. We need to fix the test framework. @yaooqinn can you help with this later?

Reply:
OK, I noticed this too

Reply:
it could be hacky to just fix the test side. I guess we should do this after the char could be a return type.

And this is not just a test fix but also a minor issue for the  `spark-sql` shell